doc/api/worker_threads.md

```diff
@@ -48,6 +48,9 @@ if (isMainThread) {
 The above example spawns a Worker thread for each `parse()` call. In actual
 practice, use a pool of Workers instead for these kinds of tasks. Otherwise, the
 overhead of creating Workers would likely exceed their benefit.
+When implementing a worker pool, it is strongly recommended to use the
+[`AsyncResource`][] API to inform diagnostic tools (for e.g. asynchronous stack
```

Comment:
Typo: `for e.g.` should be either `for example` or `e.g.`, although I would go with `such as`:

```suggestion
[`AsyncResource`][] API to inform diagnostic tools (such as asynchronous stack
```

Reply:
@trott I don’t think `for example` or just `e.g.` would work here, as asynchronous stack traces aren’t diagnostic tools by themselves, but rather provided by them. I’ve changed this to `e.g. in order to provide asynchronous stack traces`, maybe that works for you?

Reply:
Yes, that's great.

Reply:
(Total ignorable nit: Maybe drop "in order"?)